litebird_sim/scanning.py

```diff
@@ -0,0 +1,399 @@
+# -*- encoding: utf-8 -*-
```

Comment:
In my experience it always turned out to be very useful to allow arrays as inputs and follow the numpy broadcast rules. What do you think about it? I seems to me that the current API does not allow for that. I provide some examples below